cmd/examples/user-share/data_blob.json

```diff
@@ -0,0 +1 @@
+{"keys":[{"pubkey":"AwwcoVse6H5cSTuF1PLbaxO8Oq22H3r1uErTBFEHStUA","ephemeralPubkey":"A7KYpIGibwkvbHbCKhG6cCX1x/xtqU9P8p/MwEeIAP/g","encryptedDataKey":"dC9HuaJMUI/Kl8WjJZI2m+CskkekrNSmNg4/eogZXcE=","iv":"PUEDW/B2Ws+r0bzMIB9GQw=="},{"pubkey":"AjfcwSfNmMQID7uC8Ujku0KBFTJ2rPpxizNmt0xKEDm7","ephemeralPubkey":"A+jcO6YdMvP9R3HU9857BP1nFyqfgnhudh788VO3mKdg","encryptedDataKey":"qwdzetvSWcLQEP14f0tDiOCZL6ZtjRsb2usqz/NQUMc=","iv":"BKbjNfIJYDAKxJHn9Cy43w=="}],"dataLen":25,"iv":"EQQd0oRm2p1HCMbdj8y8Ww==","encryptedDataHash":"GL5PP/VfumSfPdIftxyD3ozpDgRoEcTKfqFGeyLJ5LA=","encryptedData":"aMlMZxhpCMNiya65urpXX6+7vLGeIXX58K/Rm98sGG4="}
```

Comment:
Looking at the serialized data, maybe let's modify `proto/ds/dsdata.proto` and change the following

 ```
# from
  repeated HiddenDataKey keys  = 1;

# to

  repeated HiddenDataKey hiddenDataKeys  = 1;

```